iXBRLViewerPlugin/xhtmlserialize.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2019 Workiva Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from lxml import etree
+import re
+
+class XHTMLSerializer:
+
+    selfClosableElements = ('area', 'base', 'br', 'col', 'hr', 'img', 'input', 'link', 'meta', 'param')
```

Comment:
Out of curiosity, where did this lit come from?  It doesn't seem to exactly match the [list of empty elements](https://www.w3.org/TR/html401/index/elements.html) in the HTML 4 spec.

Reply:
A random webpage that sounded convincing :-) Looks like the differences are "basefont" and "isindex" which are deprecated, and "frame".  I've added them in ad72c68 so that we're canonical.

Reply:
Thanks for the update.  I had approved since none of those matter to me :-), but perhaps they matter to someone.